frontend/src/v4/routes/components/autosuggestField/autosuggestField.component.tsx

```diff
@@ -101,7 +102,9 @@ export class AutosuggestField extends PureComponent<IProps, IState> {
 				{...options.containerProps}
 				style={{ width: this.popperNode ? this.popperNode.clientWidth : null }}
 			>
-				{options.children}
+				<ConditionalV5OrViewerScrollArea autoHeight autoHide={false}>
```

Comment:
The autoHide={false} doesn't appear to be working here

Reply:
It works for me     

https://user-images.githubusercontent.com/43547703/224287085-737480fe-39b9-499f-bd34-89d6b112d34f.mp4

Reply:
That video shows that autoHide is set to true though. Changing false to true in this line doesn't change the behaviour of the scrollbar. I think it might be because autoHide is set in the attrs of ConditionalV5OrViewerScrollAreaBase so this overrides it.

Either way, if this is the desired behaviour just omit this prop as it is true by default.

Reply:
My bad!